packages/integrations/mdx/test/mdx-plugins.test.js

```diff
@@ -96,7 +96,7 @@ describe('MDX plugins', () => {
 	it('ignores string-based plugins in markdown config', async () => {
 		const fixture = await buildFixture({
 			markdown: {
-				remarkPlugins: [['remark-toc']],
+				remarkPlugins: [['remark-toc', {}]],
```

Comment:
The tuple needs two values or it won't pass config validation. This should be what users have to do right now too it seems.

Reply:
Is it a change in the public APIs?

Reply:
No it shouldn't be. Currently if I use the same configuration in the `basics` example, I get

```
10:06:40 PM [astro] Unable to load /Users/bjorn/Work/oss/astro/examples/basics/astro.config.mjs

[config] Astro found issue(s) with your configuration:
  ! markdown.remarkPlugins.0  Invalid input.
```

It appears now because the tests will run the zod schema validation for each fixture options.